interview-questions.md

```diff
@@ -7,60 +7,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: From my knowledge, Git is a program you install into your computer to work on different parts/branches of the same project and transfer the code to and from your computer and anybody else who is doing the project. Git is a program that runs through your computer and can upload/download code to/from a cloud while github is a website and is the said cloud.
 
-  Researched answer:
+  Researched answer: Git is a developer tool used to deal with projects ranging from small to very large. Git can be used to track changes made in different branches of code and allow multiple people to work on the same projects together. The difference between Git and Github is, git is a control system that will allow you to track of your branches you are working on while github is the holder of all your code and creates save points every time you upload to your cloud. You will be able to visit these past points at any given time to check on your work.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: asking 2 values are equal will return a Booleon value.
 
-  Researched answer:
+  Researched answer: By using if/else if conditions you can true or false depending on the outcome.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: An index is only 1 character of what is displayed while a value is all of what is being displayed.
 
+  Researched answer: An index is a character that is specifically called upon in a string or array. The difference between a value and an index is a value is used to define properties, like, defining the string or number of a variable while an index is a character that is specifically called on within a string.
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer: Built-in methods are tools a program has to help the user complete different tasks quickly
 
-  Researched answer:
+  Researched answer: Built-in methods are functions coded into web pages to do tasks for the user efficiently.
```

Comment:
Almost, they are built out pieces of functionality that are made by other developers to help us save time and lines of code!